subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/core_dependency_management.adoc

```diff
@@ -14,36 +15,49 @@ For an overview of dependency management terms, see <<dependency_management_term
 [[sec:dependency-mgmt-in-gradle]]
 == Dependency Management in Gradle
 
-[.inset]
+.Dependencies management at a glance
 image::dependency-management-resolution.png[]
 
 Gradle has built-in support for dependency management.
 
-We’ll explore the main concepts with the help of the theoretical but common project to define terms:
+Let's explore the main concepts with the help of a theoretical but common project:
 
 * This project builds Java source code.
-* Some of the Java source files import classes from link:https://github.com/google/guava[Google Guava], a open-source library.
+* Some Java source files import classes from the link:https://github.com/google/guava[Google Guava] library.
 * This project uses link:http://junit.org/junit5/[JUnit] for testing.
 
-This project uses Guava and JUnit as _dependencies_.
-Gradle fetches dependencies from _repositories_.
-You can <<declaring_repositories.adoc#declaring-repositories,declare repositories>> to tell Gradle where to fetch dependencies.
-Repositories offer dependencies in multiple formats.
-For information about the formats supported by Gradle, see <<declaring_dependencies.adoc#sec:dependency-types,dependency types>>.
+The Gradle build file might look as follows:
 
-A build script developer can <<declaring_dependencies.adoc#declaring-dependencies,declare dependencies>> for different scopes.
-For example, Gradle could use one scope to compile source code and another to execute tests.
-Gradle calls the <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,scope of a dependency>> a _configuration_.
+.Gradle build file with dependencies
+====
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/kotlin",files="build.gradle.kts"]
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/groovy",files="build.gradle"]
+====
 
-During a build, Gradle locates the dependencies needed for the requested tasks.
-The dependencies might need to be downloaded from a remote repository, retrieved from a local directory, or (in a multi-project setting) built from another project.
-This process is called _dependency resolution_.
-For more information, see <<dependency_resolution.adoc#sec:how-gradle-downloads-deps,How Gradle downloads dependencies>>.
+<1> *Define repositories for project.*
```

Comment:
I find myself wanting to add some sort of implied prefix to these bullet points, but it doesn't work consistently for all points.  For instance I might expect:

_Here we_ Define repositories for the project.
OR
_This is how you would_ Define repositories for the project.

But this doesn't work well with the second bullet point "Remote and local repositories for dependency locations" which I would prefix with something like _this is an example of_ or _this is how you would configure_.

I wonder if we could make these read more consistently in a single voice?  e.g.

_Here we..._
Define all repositories for the project.
Declare each remote and local repository for dependency locations.
OR
_This is how you would configure..._
The container of all repositories of the project
An individual remote or local repository for dependency locations

Reply:
Fixed in final commit.